.github/workflows/README.md

```diff
@@ -0,0 +1,23 @@
+# CI Workflows
+
+## Overview
+
+- [Build](build.yml)
+  - Build and test Haskell code
+  - Build Docker image
+- [Draft](draft.yml)
+  - Create a GH draft release with a static binary
+- [Release](release.yml)
+  - Upload the Docker image ghcr.io
+  - Upload the package and docs to Hackage
+
+## Events
+
+```mermaid
+sequenceDiagram
+  participant Build
+  participant Draft
+  participant Release
+  Build->>Draft: Tag created
+  Draft->>Release: Publish release
+```
```

Comment:
The diagram isn't a 100% clear for me. I understand that the arrows going to the right mean moving onto those CI workflows once the event in the arrow happens. However, I'm confused by the fact that the workflows are repeated at the top and at the bottom of the diagram; and to nitpick some more, it feels like all steps are starting if you start reading the diagram from the top, and then during the build step if a tag is created, we move to the draft step mid-execution, which is quite different. I'm thinking something along these lines might be clearer to what we're trying to convey:
```suggestion
```mermaid
  flowchart LR
      Build --> tag{{Is a tag created on GH?}} -- Yes --> Draft
      tag -- No --> endci[End CI]
      Draft --> release{{Is a release published on GH?}} -- Yes --> Release
      release -- No --> endci
      Release --> endci
```

This is how that diagram would look:
<img width="1092" alt="Screenshot 2022-06-09 at 11 39 50" src="https://user-images.githubusercontent.com/22799724/172817227-596ef372-0f28-4884-883a-644e9b02d310.png">

Let me know your thoughts on this matter 🙂

Reply:
@DavidMazarro I really like this diagram! Thank you for the suggestion